[2024-03-21 17:46:08,448] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_rf_score 2024-03-21T17:45:00+00:00 [queued]>
[2024-03-21 17:46:08,490] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_rf_score 2024-03-21T17:45:00+00:00 [queued]>
[2024-03-21 17:46:08,490] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 17:46:08,490] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2024-03-21 17:46:08,490] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 17:46:08,545] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): compute_rf_score> on 2024-03-21T17:45:00+00:00
[2024-03-21 17:46:08,561] {standard_task_runner.py:52} INFO - Started process 27039 to run task
[2024-03-21 17:46:08,571] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'compute_rf_score', '2024-03-21T17:45:00+00:00', '--job-id', '9928', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpdyfbji8j', '--error-file', '/tmp/tmphxwvc3gj']
[2024-03-21 17:46:08,572] {standard_task_runner.py:77} INFO - Job 9928: Subtask compute_rf_score
[2024-03-21 17:46:08,643] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.compute_rf_score 2024-03-21T17:45:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-21 17:46:08,729] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=compute_rf_score
AIRFLOW_CTX_EXECUTION_DATE=2024-03-21T17:45:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-21T17:45:00+00:00
[2024-03-21 17:46:11,927] {python.py:151} INFO - Done. Returned value was: None
[2024-03-21 17:46:11,938] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=compute_rf_score, execution_date=20240321T174500, start_date=20240321T174608, end_date=20240321T174611
[2024-03-21 17:46:11,971] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-03-21 17:46:11,995] {local_task_job.py:151} INFO - Task exited with return code 0
